src/modules/DetectionSensorModule.cpp

```diff
@@ -46,15 +46,15 @@ int32_t DetectionSensorModule::runOnce()
     if ((millis() - lastSentToMesh) >= getConfiguredOrDefaultMs(moduleConfig.detection_sensor.minimum_broadcast_secs) &&
         hasDetectionEvent()) {
         sendDetectionMessage();
-        return moduleConfig.detection_sensor.minimum_broadcast_secs;
+        return getConfiguredOrDefaultMs(moduleConfig.detection_sensor.minimum_broadcast_secs);
```

Comment:
probably the minimum of moduleConfig.detection_sensor.minimum_broadcast_secs and moduleConfig.detection_sensor.state_broadcast_secs ?